packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/simulationservice/interventions/InterventionPolicy.java

```diff
@@ -40,4 +42,45 @@ public InterventionPolicy clone() {
 		}
 		return clone;
 	}
+
+	// Check each intervention this policy contains
+	// If any of them are are invalid the entire policy is invalid.
+	public Boolean validateInterventionPolicy() throws Exception {
+		for (int i = 0; i < this.interventions.size(); i++) {
```

Comment:
Similarly here, if you start at 1 for `i` you can drop the entire second for loop and use index `i-1`

Reply:
Every intervention needs to be checked with each other. For example the following should fail as the first and last both contain "A"
Intervention One: [A, B, C]
Intervention Two: [D, E, F]
Intervention Three[A, G, H]

Here is a picture of what this looks like with the current UI
![image](https://github.com/user-attachments/assets/9d59703c-f500-4531-9609-3c423a798a74)